docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -330,6 +327,44 @@ On touch devices, TestCafe emulates touch events instead of mouse events.
 
 Mouse event | Touch event
 ----------- | -------------
-`mousemove` (when hovering or dragging) | `touchmove` (dragging only)
+`mousemove` (when hovering or dragging) | `touchmove` (when dragging)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Interaction Requirements
+
+TestCafe actions can interact with elements if they satisfy the following conditions:
+
+* an element has to be within the page's [`body`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) or in an [`<iframe>`](#work-with-iframes). The element may be outside of a browser window- as long as TestCafe can reach it by scrolling.
+
+* an element has to be visible - its properties should be as follows:
+
+    Property | Value
+    -------- | --------
+    `display`  | *not* set to `none`
+    `visibility` | set to `visible` (the default value)
+    `width`    | auto calculated as or set to > `0`
+    `height`   | auto calculated as or set to > `0`
+
+* a targeted part of the element has to be visible.  
+
+    TestCafe actions target a center of an element, or a point specified by the `offsetX` and `offsetY` options of an action. If another element obstructs the target point, the action still executes. The [.click](../../reference/test-api/testcontroller/click.md) action, for example, affects an overlapping element instead of a targeted one.
```

Comment:
The word `still` is also not quite apt for technical docs. It usually indicates the part with `still` can be shortened or omitted:

*...If another element obstructs the target point, the action is executed with the overlapping element (for instance, the [t.click](../../reference/test-api/testcontroller/click.md) action clicks an overlapping block).*